bool acc_dec
bool emergency
float64 steer_angle
float64 obstacles_dist
float64 dist_to_goal
